layer/compile_time_layer.cc

```diff
@@ -28,6 +29,12 @@ namespace {
 // Layer book-keeping information
 // ----------------------------------------------------------------------------
 
+using time_point = std::chrono::system_clock::time_point;
+
+// A function alias used to get the current time_point from chrono and replaces
+// absl::now().
+auto Now = std::chrono::system_clock::now;
```

Comment:
nit: `constexpr auto Now`? Or make it a function, this will make it easier to tell what the return type is and it's not much more typing:
```
time_point Now() { return ...; }
```